.jshintrc

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Also, one more thing - is it typical to include this default .jshintrc file in repositories? I'm not familiar with the conventions here, but happy to learn.